src/assets/build_assets.rs

```diff
@@ -221,6 +223,20 @@ fn build_minimal_syntax_sets(
     let (other_syntax_lookup, syntax_to_dependencies, syntax_to_dependents) =
         generate_maps(syntaxes);
 
+    if let Ok(dot_file_path) = std::env::var("BAT_SYNTAX_DEPENDENCIES_TO_GRAPHVIZ_DOT_FILE") {
+        // To genarate a Graphviz dot file of syntax dependencies, do this:
```

Comment:
genarate -> generate (I will wait with updating PR until first review round is done)

Reply:
had to fix CI, so I fixed this typo too